doc/examples/segmentation/plot_expand_labels.py

```diff
@@ -0,0 +1,54 @@
+"""
+==========================================
+Expand segmentation labels without overlap
+==========================================
+
+When you have a set of regions
+segmentations. The :py:func:`skimage.segmentation.join_segmentations`
+function computes the join of two segmentations, in which a pixel is
+placed in the same segment if and only if it is in the same segment in
+*both* segmentations.
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from skimage.filters import sobel
+from skimage.measure import label
+from skimage.segmentation import watershed, expand_labels
+from skimage.color import label2rgb
+from skimage import data
+
+coins = data.coins()
+
+# Make segmentation using edge-detection and watershed.
+edges = sobel(coins)
+
+# Identify some background and foreground pixels from the intensity values.
+# These pixels are used as seeds for watershed.
+markers = np.zeros_like(coins)
+foreground, background = 1, 2
+markers[coins < 30.0] = background
+markers[coins > 150.0] = foreground
+
+ws = watershed(edges, markers)
+seg1 = label(ws == foreground)
+
+expanded = expand_labels(seg1, distance=10)
+
+# Show the segmentations.
+fig, axes = plt.subplots(nrows=1, ncols=2, figsize=(9, 5),
+                         sharex=True, sharey=True)
+
+color1 = label2rgb(seg1, image=coins, bg_label=0)
+axes[0].imshow(color1)
+axes[0].set_title('Sobel+Watershed')
+
+color2 = label2rgb(expanded, image=coins, image_alpha=0.5, bg_label=0)
```

Comment:
Ah, here's the issue, image_alpha should be removed. Will do that now.

Reply:
Actually: I'll do that after I see if the build works.

Reply:
Thanks, so it was actually related to our example.